cxx/src/lib/io/fileio.cc

```diff
@@ -83,6 +84,88 @@ long int fwrite_to_file(Seismogram& d, const string dir,const string dfile)
 
 	return(foff);
 }
+std::vector<long int> fwrite_to_file(mspass::seismic::Ensemble<mspass::seismic::TimeSeries>& d, const std::string dir,const std::string dfile)
+{
+	try{
+		FILE *fp;
+		vector<long int> foffs;
+		string fname;
+		if(dir.length()>0)
+		  /* for expected context for use in python we will assume dir does not
+			have a trailing path separator so we always insert / */
+			fname = dir + "/" + dfile;
+		else
+		  /* Null as always in unix means use current directory*/
+			fname=dfile;
+		if((fp=fopen(fname.c_str(),"a")) == NULL)
+		  /* use the name of the overloaded parent instead of the actual function - intentional*/
+			throw MsPASSError("fwrite_to_file:  Open failed on file "+fname,ErrorSeverity::Invalid);
+	  /* Both fseek and ftell can fail in weird circumstances, but I intentionally
+		do not trap that condition as if either have issues I am quite sure
+		the fwrite will fail */
+		fseek(fp,0L,2);
```

Comment:
This call to fseek is not necessary.   The file is opened "a" which means it will be position at end of file if it already exists.   The comment should be removed as well.